bundle/tests/scorecard/kuttl/day2operation/05-delete-operations.yaml

```diff
@@ -0,0 +1,16 @@
+# Need to manually delete Runtime Operations as kuttl does not automatically delete these at the end of the test
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+delete:
+  - apiVersion: rc.app.stacks/v1beta2
+    kind: RuntimeOperation
+    metadata:
+      name: example-runtime-operation-0
```

Comment:
This should be at the top level, similar to https://github.com/application-stacks/runtime-component-operator/blob/353c8e9189b065560d6ceb4bad8c881f5b12a694/bundle/tests/scorecard/kuttl/route-certificate/03-delete-secret.yaml#L8

Otherwise, kuttl deletes all RuntimeOperation - which is fine in this case. In that case, we would want
```
  - apiVersion: rc.app.stacks/v1beta2
    kind: RuntimeOperation
```